app/data/authors.yaml

```diff
@@ -93,3 +93,14 @@
     Marit is a Software Developer, a Java Champion and works as a Developer Advocate at JetBrains.
     She speaks at international conferences (including JavaZone, Devoxx, Voxxed Days, JSpring, JFall, JFokus and GOTO), local JUGs, podcasts and webinars, 
     and has contributed to the book “97 Things Every Java Programmer Should Know” (O’Reilly Media).
+
+- name: Connor Schweighöfer
+  email: squidxtv@gmail.com
+  photo_url: https://squidxtv.me/img/Connor.png
+  github: SquidXTV
+  twitter: SquidXTV
+  website: https://squidxtv.me/
+  description: |
+    Connor is a Java Developer and high school student from Germany with four years of programming experience.
```

Comment:
This isn't really any of my business but I'm still warning you about the following: "four years of programming experience" will likely get outdated.

Reply:
Ok yeah, going to remove it